configure

```diff
@@ -274,10 +308,28 @@ gen_config() {
 
 
 	# early validation
-	NODEPROPS=( $(doctl_size_list_json | jq -r '.[] | select(.slug == "'"$SIZE"'") | [.vcpus, .memory, .price_hourly] | @tsv ') )
-	NODE_CPU=${NODEPROPS[0]}
-	NODE_MEMORY=${NODEPROPS[1]}
-	NODE_PRICE=${NODEPROPS[2]}
+	if [ $PROVIDER == "do" ]; then
+	  NODEPROPS=( $(doctl_size_list_json | jq -r '.[] | select(.slug == "'"$SIZE"'") | [.vcpus, .memory, .price_hourly] | @tsv ') )
```

Comment:
Indentation is off here -- tab followed by two spaces (should be two tabs instead).

There are a few more similar indentation issues in the file that should be fixed.

Reply:
Should be fixed